dffml/source/source.py

```diff
@@ -29,12 +29,45 @@ def __init__(self, parent: "BaseSource") -> None:
     async def update(self, record: Record):
         """
         Updates a record for a source
+
+        Examples
+        --------
+        >>> async def main():
+        ...     async with ExampleSource(records=[]) as testSource:
+        ...         # Open, update, and close
+        ...         async with testSource() as sourceContext:
+        ...             example = Record("one")
+        ...             # ... Update one into our records ...
```

Comment:
We should probably call `record.evaluated` here.

Reply:
Evaluated will cause records.export() to leave a last updated timestamp here, do we want that? I'm not sure how to include timestamps in sphinx doctests but I can surely figure something out if that's the case, if not I can just add features through the initializer.

Reply:
Ah yes, the timestamps... Maybe we should just call `record.features()` then.